iguana/main.c

```diff
@@ -2257,7 +2257,7 @@ void iguana_main(void *arg)
         }
         else if ( strncmp((char *)arg,"notary",strlen("notary")) == 0 ) // must be second to last
         {   /* mainnet */
-            myinfo->rpcport = IGUANA_NOTARYPORT;
+            myinfo->rpcport = IGUANA_NOTARYPORTTESTNET;
```

Comment:
This is the path for mainnet. 

Change it here
https://github.com/KomodoPlatform/dPoW/blob/dce86ff705ca4887db28cbdbae25bad4c6d46a8d/iguana/main.c#L2268